### DISCUSSION TITLE
New UI Based Theme Editor

### DISCUSSION THREAD

**Discussion by ykoitzsch** [primefaces/community#1064] [PrimeNG] [answered]
Hello, 

according to the roadmap [here](https://primeng.org/roadmap), the "**New UI Based Theme Editor**" should have been released by now and I was wondering about its progress and when we can expect it. I am looking forward to it :)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

The development of the new UI-based Theme Editor is still ongoing, and the roadmap will be updated, we are sorry for the latency.
